modules/internal/ChapelTaskData.chpl

```diff
@@ -101,6 +102,26 @@ module ChapelTaskData {
     return v == 1;
   }
 
+  proc chpl_task_data_setNextCoStmtSerial(tls:c_ptr(chpl_task_infoChapel_t), makeSerial: bool) : void {
+    var prv = tls:c_ptr(c_uchar);
+    var i = chpl_offset_nextCoStmtSerial;
+    var v:uint(8) = 0;
+    if makeSerial then
+      v = 1;
```

Comment:
Pretty sure you could write `var v = makeSerial:uint(8)` for all that

Reply:
I copied `chpl_task_data_setSerial`

Reply:
Not that I disagree with your point, but I was striving to be consistent and figured there might be some reason it wasn't just casting (possibly because these things are used early in compilation and maybe we haven't resolved casts yet?)

Reply:
That is possible and it's not a big deal in any case.